src/Registration/pReg/Reg.py.in

```diff
@@ -237,6 +237,11 @@ class NiftiImageData(SIRF.ImageData):
         """Get sum."""
         return parms.float_par(self.handle, 'NiftiImageData', 'sum')
 
+    def dimensions(self):
+        """Returns image dimensions as a tuple (nz, ny, nx)."""
+        dim  = self.get_dimensions()
+        return tuple(dim[1:1+dim[0]]) # dim[0] tells us how many dimensions
+
```

Comment:
You just need an alias:
```
def dimensions(self):
    return self.get_dimensions()
```

Reply:
There is a reason, I wrote it on the other comment and on the commit in master.

Reply:
I looked through your comments but still cannot see why you need `dimensions()` that returns different value from `get_dimensions()`? Do you use `dimensions()` anywhere else (not only for size checking in `assert_validities`)?

Reply:
@evgueni-ovtchinnikov I don't need it, but `get_dimensions()` returns a different object type than the already implemented 2 or 3 `dimensions()` in SIRD. The way I coded it all different calls to `dimensions()` in sirf would return the same: a tuple.

Reply:
Oh I see, you are after uniformity here - fine, do it your way then, I do not see any problems with `get_dimensions()` returning different stuff from `dimensions()`.